pkg/sloop/server/internal/config/config_test.go

```diff
@@ -16,8 +16,9 @@ func Test_loadFromJSONFile_Success(t *testing.T) {
 	err = json.Unmarshal(configFile, &expectedconfig)
 	assert.Nil(t, err)
 
-	out_config := loadFromFile(configfilename)
-	assert.Equal(t, out_config, &expectedconfig)
+	var outConfig *SloopConfig
+	outConfig = loadFromFile(configfilename,outConfig)
+	assert.Equal(t, outConfig, &expectedconfig)
```

Comment:
can we also add some unit tests that ensure overrides are taking precedence and default values are not changed if they are not present in config/flag.

Reply:
+1

Reply:
any pointers on what way would be acceptable to u?

Reply:
any way that covers the new behavior (flags>config>default value).